configure/configure.js

```diff
@@ -0,0 +1,36 @@
+const core = require('@actions/core');
+const exec = require('@actions/exec');
+const io = require('@actions/io');
+const configure = require('../src/configure');
+const installTelepresence = require('../src/install');
+const cache = require('@actions/cache');
+
+const telepresenceConfiguring = async function () {
+    await installTelepresence.telepresenceInstall();
+
+    const path = configure.getTelepresenceConfigPath();
+    const restorePath = [path];
+
+    try {
+        await io.mkdirP(path);
+        await cache.restoreCache(restorePath, configure.TELEPRESENCE_CACHE_KEY);
+    } catch (error) {
+        core.warning(`Unable to find the telepresence id: ${error}`);
+    }
+    try {
+        await exec.exec(`${installTelepresence.TP_PATH}/telepresence`, ['connect']);
```

Comment:
curious why we need to reference the `TP_PATH` here. Is it a custom path or the tp binary will not be available in the `path` anymore?

Reply:
looks like `core.addPath` should help us there to reference `telepresence` as we did

Reply:
I have some issues calling telepresence from there, that's why I'm using the path, let's try again this and we can decide if we keep it or remove it